packages/xsnap/lib/lockdown-shim.js

```diff
@@ -1,4 +1,4 @@
 // @ts-nocheck
-import 'ses/lockdown';
+import 'ses';
```

Comment:
so this somehow no longer brings in babel?

Reply:
Yes. I factored `@endo/static-module-record` out of SES, so SES no longer depends on Babel. StaticModuleRecord, however, _does_, so if you want it, you have to buy it. Compartment Mapper is refactored so you don’t need StaticModuleRecord to import `@endo/compartment-mapper/import-archive.js`, which is the subset we need in `@agoric/import-bundle`. So, you only need SMR if you’re building Zip bundles, not when you run them.

Reply:
Please promote this explanation to a commit message, if not a code comment (is there a relevant NEWS.md?)

That is: please hoist this valuable answer to "why bother?".

In fact, perhaps the PR title should be:
 - Integrate SES 0.13, which no longer depends on Babel